Project Requirements Document.md

```diff
@@ -0,0 +1,28 @@
+|            **Project Name**            | Threewide                                                                                                                                                                                                    |
+|       **Functional Requirements**      |                                                                                                                                                                                                              |
+|            **High Priority**           | 1. Beginners can play Tetris game with modern tetris ruleset.                                                                                                                                                |
+|                                        | 2. Beginners can learn strategies to apply into game.                                                                                                                                                            |
+|                                        | 3. Our goal is to teach number of T spins through our plateform.                                                                                                                                             |
+|                                        | 4. Our plateform should provide some embedded small game tasks to play.                                                                                                                                      |
+|                                        | 5. We should be able to verify that the user accomplished that goal.                                                                                                                                         |
+|            **Low Priority**            | 1. Add more customization to the Tetris game on our plateform for beginners. For instance, we can add teaching strategies for other Tetris blocks such as super rotation system kick table, DAS and ARP etc. |
+|                                        | 2. Add some advance learning material on higher level.                                                                                                                                                       |
+| **Technical/Performance Requirements** | 1. React.js - React.js to design attractive and interactive interface for the beginners.                                                                                                                     |
+|                                        | 2. Node.js - Node.js will be used to managed backend of the Threewide. This will help us to add more embedded game and interacting features for the users.                                                   |
```

Comment:
Node will actually be used to render the pages of our platform. Providing the user with the embedded game and React app with the Tetris problem data